Sources/Mantle/Signature.swift

```diff
@@ -135,6 +135,14 @@ extension Signature {
                                             inside: newDef))
   }
 
+  func addLetBinding(_ name: QualifiedName, type: Type<TT>,
+                     _ tel: Telescope<TT>) {
+    let definition = Definition.constant(type, .function(.open))
```

Comment:
Just a thought: Maybe we should add let bindings as a new kind of constant.  Not sure it's worth the effort.